pkg/koordlet/reporter/reporter.go

```diff
@@ -139,9 +138,9 @@ func (r *reporter) Run(stopCh <-chan struct{}) error {
 			return fmt.Errorf("timed out waiting for node metric caches to sync")
 		}
 
-		go wait.Until(func() {
-			r.sync()
-		}, time.Duration(r.config.ReportIntervalSeconds)*time.Second, stopCh)
+		reportInterval := time.Duration(r.config.ReportIntervalSeconds) * time.Second
```

Comment:
Note in this way the first synchronization is forced to run after `config.ReportIntervalSeconds`. Could you please make the first round run immediately and use a custom interval.

Reply:
done